api/v1alpha1/accesslogging_types.go

```diff
@@ -120,6 +120,7 @@ type ALSEnvoyProxyAccessLog struct {
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=1
 	// +kubebuilder:validation:XValidation:message="BackendRefs only supports Service kind.",rule="self.all(f, f.kind == 'Service')"
+	// +kubebuilder:validation:XValidation:message="BackendRefs only supports Core group.",rule="self.all(f, f.group == '')"
```

Comment:
`f.group == ''` is this correct ?

Reply:
https://github.com/kubernetes/api/blob/master/core/v1/register.go#L26
core api group is '' .   it may works.

Reply:
`core` is also valid?

Reply:
core is invalid. core api has no group. so only group=='' is valid. 

Correct me . if I'm wrong，thx~

Reply:
I found some usage about `group: core` in mesh and gep-709, @arkodg anything recall?

Reply:
this looks good to me
`''` == `core` https://github.com/kubernetes-sigs/gateway-api/blob/45c7d4ff434f744b5bf55ed81dfd059d78cd33f3/apis/v1/object_reference_types.go#L97